about/licenses/index.md

```diff
@@ -67,6 +67,7 @@ is the copyright year.
 - [GPL-2.0 License](/about/licenses/gpl-2.0.txt)
 - [GPL-3.0 License](/about/licenses/gpl-3.0.txt)
 - [LGPL-2.1 License](/about/licenses/lgpl-2.1.txt)
+- [Apache 2.0 License with LLVM Exception](/about/licenses/apache-2.0-llvm-exception.txt)
```

Comment:
Seems like this should go next to the regular Apache license in the list, or maybe the list should be alphabetized?

It might be worth adding a note like:

```
[Apache 2.0 License with LLVM Exception](/about/licenses/apache-2.0-llvm-exception.txt) (compatible with GPL-2.0 and LGPL-2.1)
```